WA.Pizza.Infrastructure/Data/Services/BasketDataService.cs

```diff
@@ -18,34 +18,29 @@ public BasketDataService(AppDbContext dbContext)
 
 
 
-        public ICollection<GetBasketDTO> GetBasketItems(int basketId)
+        public ICollection<BasketDTO> GetBasketItems(int basketId)
```

Comment:
Method tell me that I can get all basket items by basket id but...

Reply:
So just GetBasket (method name)?